2025-03-17 20:56:27 error: uncaughtException: Cannot find module '../services/invitation.service'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js
- /home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
Error: Cannot find module '../services/invitation.service'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js
- /home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js:2:27)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
2025-03-17 20:56:37 error: uncaughtException: Cannot find module '../services/invitation.service'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js
- /home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
Error: Cannot find module '../services/invitation.service'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js
- /home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/controllers/invitation.controller.js:2:27)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
2025-03-17 21:01:03 error: uncaughtException: Route.post() requires a callback function but got a [object Undefined]
Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/route.js:216:15)
    at proto.<computed> [as post] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js:15:8)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
2025-03-17 21:01:09 error: uncaughtException: Route.post() requires a callback function but got a [object Undefined]
Error: Route.post() requires a callback function but got a [object Undefined]
    at Route.<computed> [as post] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/route.js:216:15)
    at proto.<computed> [as post] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/invitation.routes.js:15:8)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
2025-03-18 14:08:08 error: uncaughtException: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
Error: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:72:23)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
2025-03-18 14:08:08 error: uncaughtException: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
Error: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:72:23)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
2025-03-18 14:08:08 error: uncaughtException: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
Error: Cannot find module './routes/admin.routes'
Require stack:
- /home/temitope/Desktop/work/geofencemain/backend/src/app.js
- /home/temitope/Desktop/work/geofencemain/backend/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:72:23)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
2025-03-18 14:08:41 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:86:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:08:41 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:86:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:08:46 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:86:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:09:50 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:86:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:10:21 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:87:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:10:22 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:87:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:10:51 error: uncaughtException: authenticate is not defined
ReferenceError: authenticate is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:88:5)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 14:50:23 error: uncaughtException: PlatformController is not a constructor
TypeError: PlatformController is not a constructor
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/platform.routes.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:21:24)
2025-03-18 14:56:49 error: uncaughtException: PlatformController is not a constructor
TypeError: PlatformController is not a constructor
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/platform.routes.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:21:24)
2025-03-18 14:58:32 error: uncaughtException: PlatformController is not a constructor
TypeError: PlatformController is not a constructor
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/platform.routes.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:21:24)
2025-03-18 14:59:06 error: uncaughtException: PlatformController is not a constructor
TypeError: PlatformController is not a constructor
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/platform.routes.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:21:24)
2025-03-18 20:06:52 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/route.js:216:15)
    at proto.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/systemAdmin/v1/platform.routes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
2025-03-18 20:07:06 error: uncaughtException: platformRoutes is not defined
ReferenceError: platformRoutes is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:72:26)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 20:08:39 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/route.js:216:15)
    at proto.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/systemAdmin/v1/platform.routes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
2025-03-18 20:11:03 error: uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/route.js:216:15)
    at proto.<computed> [as get] (/home/temitope/Desktop/work/geofencemain/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/systemAdmin/v1/platform.routes.js:8:8)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
2025-03-18 20:57:18 error: uncaughtException: platformRoutes is not defined
ReferenceError: platformRoutes is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:72:26)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/server.js:2:13)
2025-03-18 23:42:09 error: uncaughtException: employeeController is not defined
ReferenceError: employeeController is not defined
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/employee.routes.js:12:25)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:17:24)
2025-03-18 23:42:28 error: uncaughtException: Invalid or unexpected token
/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/employee.routes.js:3
const EmployeeController = require('../../controllers/employee.controller
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1383:18)
    at Module._compile (node:internal/modules/cjs/loader:1412:20)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:17:24)
2025-03-18 23:42:45 error: uncaughtException: Invalid or unexpected token
/home/temitope/Desktop/work/geofencemain/backend/src/routes/v1/employee.routes.js:3
const EmployeeController = require('../../controllers/employee.controller
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at wrapSafe (node:internal/modules/cjs/loader:1383:18)
    at Module._compile (node:internal/modules/cjs/loader:1412:20)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Module.load (node:internal/modules/cjs/loader:1282:32)
    at Module._load (node:internal/modules/cjs/loader:1098:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/home/temitope/Desktop/work/geofencemain/backend/src/app.js:17:24)
